develop/dev-guide-gui-mysql-workbench.md

```diff
@@ -0,0 +1,152 @@
+
+In this tutorial, you can learn how to connect to your TiDB cluster using MySQL Workbench.
+
+> **Note:**
+>
+> This tutorial is compatible with TiDB Serverless, TiDB Dedicated, and TiDB Self-Hosted.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- [MySQL Workbench](https://dev.mysql.com/downloads/workbench/) **8.0.31** or later versions.
+- A TiDB cluster.
+
+<CustomContent platform="tidb">
+
+**If you don't have a TiDB cluster, you can create one as follows:**
+
+- (Recommended) Follow [Creating a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md) to create your own TiDB Cloud cluster.
+- Follow [Deploy a local test TiDB cluster](/quick-start-with-tidb.md#deploy-a-local-test-cluster) or [Deploy a production TiDB cluster](/production-deployment-using-tiup.md) to create a local cluster.
+
+</CustomContent>
+<CustomContent platform="tidb-cloud">
+
+**If you don't have a TiDB cluster, you can create one as follows:**
+
+- (Recommended) Follow [Creating a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md) to create your own TiDB Cloud cluster.
+- Follow [Deploy a local test TiDB cluster](https://docs.pingcap.com/tidb/stable/quick-start-with-tidb#deploy-a-local-test-cluster) or [Deploy a production TiDB cluster](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup) to create a local cluster.
+
+</CustomContent>
+
+## Connect to TiDB
+
+Connect to your TiDB cluster depending on the TiDB deployment option you have selected.
+
+<SimpleTab>
+<div label="TiDB Serverless">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+
+3. Ensure the configurations in the connection dialog match your operating environment.
+
+    - **Endpoint Type** is set to `Public`.
+    - **Connect With** is set to `General`.
+    - **Operating System** matches your environment.
+
+4. Click **Create password** to create a random password.
+
+    > **Tip:**
+    >
+    > If you have created a password before, you can either use the original password or click **Reset password** to generate a new one.
+
+5. Launch MySQL Workbench and click the **+** in the right side of **MySQL Connections**.
+
+    ![MySQL Workbench: add new connection](/media/develop/navicat-add-new-connection.png)
+
+6. In the setup panel, configure the following connection parameters:
+
+    - **Connection Name**: give this connection a meaningful name.
+    - **Hostname**: enter the `host` parameter from the TiDB Cloud connection dialog.
+    - **Port**: enter the `port` parameter from the TiDB Cloud connection dialog.
+    - **Username**: enter the `user` parameter from the TiDB Cloud connection dialog.
+    - **Password**: click **Store in Keychain ...**, enter the password of the TiDB Serverless cluster, then click **OK** to store the password.
```

Comment:
It Keychain available on all platforms, including Windows?

Reply:
It is: https://dev.mysql.com/doc/workbench/en/wb-mysql-connections-vault.html